tlscanary/template/report_template.htm

```diff
@@ -1,23 +1,52 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>PAGE_TITLE</title>
-  <link rel="stylesheet" href="../../css/report_styles.css">
+<head lang="en">
+  <meta charset="UTF-8">
```

Comment:
nit: empty HTML elements should be closed with `/>`, for example `<br />`.

I would still land this without, though.